toolchain/lowering/BUILD

```diff
@@ -24,8 +24,11 @@ cc_library(
     srcs = [
         "lowering_context.cpp",
         "lowering_function_context.cpp",
-        "lowering_handle.cpp",
-    ],
+    ] +
+    # Glob handler files to avoid missing anyway.
```

Comment:
```suggestion
    # Glob handler files to avoid missing any.
```

Reply:
Fixed here and in the other two BUILD files I copied from :)